operator/test/test.sh

```diff
@@ -72,7 +72,7 @@ init() {
     pwd
   )
   export KUBECONFIG
-  NAMESPACE="pipeline-service-test"
+  NAMESPACE="plnsvc-tests"
```

Comment:
Curious, why the change of namespace?

Reply:
the change in question was done in this branch before "pipeline-service-test" was set main.
And access in MVP infra is set for `plnsvc-tests`  so it is easier to put here the corresponding ns rather than ask to change the access for the pipeline-service test ns but it is possible if there are good reasons to keep the other NS name.